core/examples/core_samples_per_second.rs

```diff
@@ -33,8 +33,8 @@ pub fn main() {
             SoundfontInitOptions {
                 bank: None,
                 preset: None,
+                vol_envelope_options: Default::default(),
```

Comment:
This file was for raw samples-per-second testing with everything set to fast mode by default, so imo change this to be all linear, so we have more explicit control over individual flags